R/cr_md_fetch.R

```diff
@@ -10,7 +10,7 @@
 get_cr_md <- function(dois, .progress = "text") {
   checkmate::assert_character(dois, any.missing = FALSE, unique = TRUE)
   tt <- rcrossref::cr_works(dois = dois, .progress = .progress)[["data"]]
-  if (!is.null(tt)) {
+  if (nrow(tt) != 0) {
```

Comment:
this makes the function not length-stable, but we'll fix these APIs later.
Length-stably, `get_cr_md()` should either return a df with 0 rows, or it should throw an error if we think that it makes more sense (I'd lean towards error).
In that case, the responsibility to pass on dois which actually *have* metadata on cr falls to upstream functions (as in biblids).

Anyway, this hotfixes #113 for now.